bgpd/bgpd.h

```diff
@@ -2508,4 +2503,9 @@ void peer_tcp_mss_unset(struct peer *peer);
 
 extern void bgp_recalculate_afi_safi_bestpaths(struct bgp *bgp, afi_t afi,
 					       safi_t safi);
+
+#ifdef _FRR_ATTRIBUTE_PRINTFRR
+#pragma FRR printfrr_ext "%pBP"(struct peer *)
```

Comment:
needs a space between `"%pBP"` and `(struct peer *)`

(clang probably misformats it because it doesn't understand the semantics of this pragma)

Reply:
(add `/* clang-format off */` + `/* clang-format on */` around it if clang-format removes the space)

Reply:
Thanks!

Reply:
Fixed.